Cargo.toml

```diff
@@ -22,6 +22,7 @@ num = "0.4.0"
 num-derive = "0.3.3"
 num-traits = "0.2.14"
 nvml-wrapper = "0.7.0"
+openssl = { version = "0.10.33", features = ["vendored"] }
```

Comment:
Why do you force vendored openssl?

Reply:
It looked to be necessary to build with musl toolchain in CI on ubuntu based on some pkg-config errors about cross-compilation.